pkg/metrics/metrics.go

```diff
@@ -187,6 +192,29 @@ func (pc *PrometheusNodeConfig) Configure() error {
 	return nil
 }
 
+// createMetricsEndpoint creates an endpoint object in the operator namespace.
+// WMCO is no longer creating a service with selector hence no Endpoint object is created
+// and WMCO needs to create it.
+func (pc *PrometheusNodeConfig) createMetricsEndpoint() (*v1.Endpoints, error) {
```

Comment:
I am confused now :-) Shouldn't this be created by OLM?

Reply:
This cannot be part of OLM manifest. I was getting `invalidResource` error when trying to deploy the `Endpoints` resource.

Reply:
Please add this as a comment.